api/authn/user_store.go

```diff
@@ -48,3 +48,27 @@ func (s *userStore) Create(apiContext *types.APIContext, schema *types.Schema, d
 
 	return created, err
 }
+
+func (s *userStore) List(apiContext *types.APIContext, schema *types.Schema, opt types.QueryOptions) ([]map[string]interface{}, error) {
+
+	req := apiContext.Request
+	me := req.URL.Query().Get("me")
+
+	if me == "true" {
+		schemaData := make([]map[string]interface{}, 1)
+		data := make(map[string]interface{})
+		var err error
```

Comment:
Any specific reason we are declaring error variable?

Reply:
It's to avoid accidentally redefining `data` at L64.